src/core/bond_exist.cpp

```diff
@@ -0,0 +1,22 @@
+#include"bond_exist.hpp"
+#include"interaction_data.hpp"
+#include"bond.hpp"
+bool make_bond_exist_bond_class(int bond_type){
```

Comment:
This is not only simpler, it is also a step in the direction of abstracting away the implementation of bonds_ia, which will probably be changed in the future.